survey-runner.tf

```diff
@@ -655,6 +487,7 @@ module "survey-runner-vpc" {
   env                 = "${var.env}"
   aws_account_id      = "${var.aws_account_id}"
   aws_assume_role_arn = "${var.aws_assume_role_arn}"
+  vpc_name            = "eq"
```

Comment:
This may be problematic when we comes to PreProd / Prod for the runner side as it will rename the VPC.
Im not sure if a rename of a VPC requires a destroy / create.

Reply:
Might be worth seeing how the `plan` looks before applying?

Reply:
Just tested this, the renaming of the VPC is fine.
Its the db subnet group that causes issues, but seeing as that already has `eq` in it if we call the vpc `eq` it will be fine.

Reply:
Cool, so no changes necessary right now?

Reply:
no changes 👍